build/buildBuildImages.sh

```diff
@@ -125,14 +127,16 @@ touch $ACR_BUILD_IMAGES_ARTIFACTS_FILE
 echo
 echo "-------------Creating slim build image-------------------"
 buildDockerImage "$BUILD_IMAGES_SLIM_DOCKERFILE" \
-				"$ACR_SLIM_BUILD_IMAGE_REPO" \
+                "slim" \
+				"$ACR_BUILD_IMAGES_REPO" \
 				"$ORYXTESTS_SLIM_BUILDIMAGE_DOCKERFILE" \
-				"$ORYXTESTS_SLIM_BUILDIMAGE_REPO" \
-				"$DEVBOX_SLIM_BUILD_IMAGE_REPO" 
+				"$ORYXTESTS_BUILDIMAGE_REPO" \
+				"$DEVBOX_BUILD_IMAGES_REPO"
 
 echo
 echo "-------------Creating full build image-------------------"
 buildDockerImage "$BUILD_IMAGES_DOCKERFILE" \
+                "latest" \
```

Comment:
We should pass in empty here and handle in above function?

Reply:
Rearranged the arguments of this function to make the base tag last and handled `latest` in the function itself.